src/server.py

```diff
@@ -483,9 +492,13 @@ def _bus_name_disconnect_cb(self, unused_connection, bus_name):
         del self._watcher_by_bus_name[bus_name]
 
     def _do_overlap_behaviour(self, sound_event_id, overlap_behavior):
-        uuid_ = self._uuid_by_event_id.get(sound_event_id)
-        if uuid_ is None:
+        if overlap_behavior == "overlap":
             return None
+        uuids = self._uuid_by_event_id.get(sound_event_id)
+        assert len(uuids) <= 1
+        if len(uuids) == 0:
+            return None
+        uuid_ = next(iter(uuids))
```

Comment:
If you're sure at this point that `uuids` only has one element, why do you use `next(iter(uuids))`. (Not saying it's wrong or needs to change, just curious)

Reply:
this is a set? what other way do I have to do this? `list(uuids)[0] ?

Reply:
I've used `list(uuids)[0]`. But it's fine. Was just curious like I said.